dot-studio/svc_coord

```diff
@@ -51,42 +51,75 @@ function wait_for_svc_to_load() {
 }
 
 document "wait_for_port_to_listen" <<DOC
-  Wait for a port to be listening.
-
+  Wait for a port to be listening. If the port is not found in the set amount of time the function exits with 1
   @(arg:1) Port to wait for to be listening
-  @(arg:2) Wheter or not this process runs silently
+  @(arg:2) Number of seconds to wait to exit with 1. Default 60 seconds.
+  @(arg:3) Wheter or not this process runs silently
```

Comment:
I'm not so sure about modifying the arguments order. I'm sure there are places we do:
```bash
wait_for_port_to_listen 9200 silently
```

If that's the case, what will it happen? 🤔  Are we going to explode trying to do:
```bash
# If SECONDS_WAITING="silently" then ???
+    if [ $COUNTER -ge $SECOUNDS_WAITING ]
```